addons/tmf_loadouts/loadouts/US_USMC_2003.hpp

```diff
@@ -0,0 +1,533 @@
+	backpackItems[] += {"ACE_MapTools"};
+	linkedItems[] += {
+		"ACE_Vector",
+		"ItemGPS"
+	};
+};
+
+class hmgg : car {
+	displayName = "HMG Gunner";
+	backPack[] = {"CUP_B_M2_Gun_Bag"};
+};
+
+class hmgac : car {
+	displayName = "HMG Ammo Carrier";
+	backPack[] = {"CUP_B_M2_MiniTripod_Bag"};
+};
+
+class hmgtl : car {
+	displayName = "HMG Team Leader";
+	backPack[] = {"CUP_B_M2_Tripod_Bag"};
+	linkedItems[] += {
+		"ACE_Vector",
+		"ItemGPS"
+	};
+};
+
+class matg : car {
+	displayName = "MAT Gunner";
+	backpack[] = {"usm_pack_alice"};
+	secondaryWeapon[] = {"CUP_launch_Mk153Mod0"};
+	secondaryAttachments[] = {"cup_optic_smaw_scope"};
+	magazines[] += {
+		LIST_2("CUP_SMAW_HEAA_M")
+	};
+};
+
+class matac : car {
+	displayName = "MAT Ammo Carrier";
+	backpack[] = {"usm_pack_alice"};
+	backpackItems[] = {
+		LIST_2("CUP_SMAW_HEAA_M")
+	};
+};
+
+class mattl : car {
+	displayName = "MAT Team Leader";
+	backpack[] = {"usm_pack_alice"};
+	linkedItems[] += {
+		"ACE_Vector",
+		"ItemGPS"
+	};
+	backpackItems[] = {
+		"CUP_SMAW_HEAA_M",
+		"ACE_MapTools"
+	};
+};
+
+class hatg : car {
+	displayName = "HAT Gunner";
+	backpack[] = {"usm_pack_alice"};
+	secondaryWeapon[] = {"CUP_launch_Javelin"};
+	magazines[] += {"CUP_Javelin_M"};
+};
+
+class hatac : car {
+	displayName = "HAT Ammo Carrier";
+	backpack[] = {"usm_pack_alice"};
+	backpackItems[] = {"CUP_Javelin_M"};
+};
+
+class hattl : car {
+	displayName = "HAT Team Leader";
+	backpack[] = {"usm_pack_alice"};
+	linkedItems[] += {
+		"ACE_Vector",
+		"ItemGPS"
+	};
+	backpackItems[] = {
+		"CUP_Javelin_M",
+		"ACE_MapTools"
+	};
+};
+
+class mtrg : car {
+	displayName = "Mortar Gunner";
+	secondaryWeapon[] = {"ace_csw_staticMortarCarry"};
+	backPack[] = {"usm_pack_alice"};
+	backpackItems[] = {
+		LIST_4("ACE_1Rnd_82mm_Mo_HE"),
+		"ACE_1Rnd_82mm_Mo_Illum",
+		"ACE_1Rnd_82mm_Mo_Smoke"
+	};
+};
+
+class mtrac : car {
+	displayName = "Mortar Ammo Carrier";
+	backPack[] = {"usm_pack_alice"};
+	backpackItems[] = {
+		LIST_4("ACE_1Rnd_82mm_Mo_HE"),
+		"ACE_1Rnd_82mm_Mo_Illum",
+		"ACE_1Rnd_82mm_Mo_Smoke"
+	};
+};
+
+class mtrtl : car {
+	displayName = "Mortar Team Leader";
+	secondaryWeapon[] = {"ace_csw_carryMortarBaseplate"};
+	items[] = { "MTR_GEAR" };
+	backPack[] = {"usm_pack_alice"};
+	backpackItems[] = {
+		LIST_4("ACE_1Rnd_82mm_Mo_HE"),
+		"ACE_1Rnd_82mm_Mo_Illum",
+		"ACE_1Rnd_82mm_Mo_Smoke"
+	};
+
+	linkedItems[] += {
+		"ACE_Yardage450",
+		"ItemGPS"
+	};
+};
+
+class samg : car {
+	displayName = "AA Missile Specialist";
+	secondaryWeapon[] = {"CUP_launch_FIM92Stinger"};
+};
+
+class samag : car {
+	displayName = "AA Assistant Missile Specialist";
+	secondaryWeapon[] = {"CUP_launch_FIM92Stinger"};
+	linkedItems[] += {
+		"Binocular",
+		"ItemGPS"
+	};
+};
+
+class sn : r {
+	displayName = "Sniper";
+	headgear[] = {"CUP_H_USMC_BOONIE_PRR_DES"};
+	uniform[] = {"CUP_U_B_USMC_FROG1_DMARPAT"};
+	goggles[] = {"default"};
+	vest[] = {"CFP_Alice_Vest2"};
+	primaryWeapon[] = {"CUP_srifle_M40A3_bipod"};
```

Comment:
Only use preset rifles for backpacks and such. Otherwise always use scope[], bipod[] and attachment[].